gradio/__init__.py

```diff
@@ -225,6 +225,7 @@
     "close_all",
     "deploy",
     "get_package_version",
+    "js",
```

Comment:
nit: looks like this is actually not exported from this module?

> "js" is specified in __all__ but is not present in modulePylance[reportUnsupportedDunderAll](https://github.com/microsoft/pyright/blob/main/docs/configuration.md#reportUnsupportedDunderAll)

Reply:
thanks fixed @whitphx